### ISSUE TITLE
CopyCat Scam Website - What to do?

### ISSUE THREAD

**Issue by patrickwolf** [iancoleman/bip39#658]
It looks like this site copied your code and instead of generating random Mnemonic codes it asks users to enter their real ones: https://www.bip32.net/ and it doesn't offer an offline version. 

My concern is that it's setup for scamming purposes and uses the official url bip32.net to get peoples private codes.

Is there awareness if the site is a scam? And if yes where to report a site like this?

Thanks

![image](https://github.com/iancoleman/bip39/assets/758811/382e4946-3113-463b-ab21-45442fa8359d)

**Comment by wigy-opensource-developer** [comment]
Indeed, it posts all private input to its backend opposed to the original code. To be honest, anyone who is careful enough would download the standalone html and run it locally.

The bip32.net domain is behind a whois anonymizer from Malaysia. There were multiple such phishing copies, some even paid Google to show up above the legitimate site.

**Comment by tauseedzaman** [comment]
careful with that

> Indeed, it posts all private input to its backend opposed to the original code. To be honest, anyone who is careful enough would download the standalone html and run it locally.
> 
> The bip32.net domain is behind a whois anonymizer from Malaysia. There were multiple such phishing copies, some even paid Google to show up above the legitimate site.

thats right

**Comment by jesobreira** [comment]
How about this one? Even the index is simlar, but conveniently they removed the links to the source codes https://www.iancoleman.net/

**Comment by wigy-opensource-developer** [comment]
> How about this one? Even the index is similar, but conveniently they removed the links to the source codes https://www.iancoleman.net/

That site reports all generated or entered secrets to some server hidden behind Cloudflare :smile: There is a really nice obfuscated JavaScript code at the beginning that makes it less obvious what format the phrase is encoded in:

```json
{"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=afgOW%2FRrowY8MVy%2BYWaAPO5zEackJVK%2B8P%2FV3pdEvoRKrhrbB9DS2yRCVshEwIuq6qN8x3PJcdk430DG%2BvdVq2uXS05veP8pqQa2%2BajEtVSFlQNA98IbBC6mt75lznF4UFPjNiI%3D"}],"group":"cf-nel","max_age":604800}
```
